### PR TITLE
[MOSIP-31019] Updated vidivi.py

### PR DESCRIPTION
**Pull request by Rakshithb1**

### CODE REVIEW

File: release/vidivi/vidivi.py [hunk 1]
```diff
@@ -157,6 +157,20 @@ def main():
...
+    print_log('*' * 50, 'info')
+    for image in images:
+        imgName = image[0][: image[0].find(":")]
+        destImgName = config['docker']['destination_organization'] + "/" + (imgName.split('/')[-1])
+        destImgTag = image[1]
+        destImgHash = getDockerHash(destImgName, destImgTag)
+        print_log("", 'info')
+        print_log("=" * 4 + " Destination Image Hash Before Push = \"" + destImgName + ":" + destImgTag + "\" IMAGE_ID : " + destImgHash, 'info')
+        print_log("", 'info')
+
```
Comment (syedsalman3753) [inline release/vidivi/vidivi.py]: Please remove this and print docker hash id in destination existence section

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: release/vidivi/vidivi.py (hunk 1 above)
Comment (Rakshithb1) [reply release/vidivi/vidivi.py]: updated as requested

**Review by syedsalman3753 — APPROVED** [review]

**Comment by ckm007** [comment]
seems your update appends the logs every time to vidivi.log, instead it should overwrite the same and raise the PR.
lets seperate logger funtion.
also the image movement is serial ow.. lets move them parallely using concurrent.futures

File: release/vidivi/vidivi.py [hunk 2]
```diff
@@ -168,10 +168,11 @@ def main():
     for image in images:
         srcImgName = image[0][: image[0].find(":")]
         srcImgtag = image[0][image[0].find(":") + 1:]
+        srcregistryUrl = "https://registry.hub.docker.com/v2/"
```
Comment (ckm007) [inline release/vidivi/vidivi.py]: take the value from variable where this value is default.

**Pull request merged**